update2 in the introduction.md (#699)
nf-core/nf-co.re: markdown/usage/introduction.md

    * The simplest way to run is with `-profile docker` (or `singularity`) which will tell Nextflow to execute jobs locally using Docker to fulfil the software requirements.

    * Conda is also supported with `-profile conda`. However this option is not recommended, as reproducibility of the results can't be guaranteed without containerization.
    

>     * If you are a member of one of the institution listed out in the [documentation](https://github.com/nf-core/configs#documentation), pleaase use the custom config file [nf-core/configs/conf](https://github.com/nf-core/configs/tree/master/conf) within your organization. 

Reviewer: ```suggestion
    * If you are a member of one of the institutions listed in the [documentation](https://github.com/nf-core/configs#documentation), please use the custom config file [nf-core/configs/conf](https://github.com/nf-core/configs/tree/master/conf) that has already been created for your institution.  
```
